.buildkite/run-cpu-test.sh

```diff
@@ -46,7 +46,7 @@ docker exec cpu-test bash -c "
 docker exec cpu-test bash -c "
   export VLLM_CPU_KVCACHE_SPACE=10 
   export VLLM_CPU_OMP_THREADS_BIND=48-92 
-  python3 -m vllm.entrypoints.openai.api_server --model facebook/opt-125m & 
+  python3 -m vllm.entrypoints.openai.api_server --model facebook/opt-125m --dtype bfloat16 & 
```

Comment:
Why do we need bfloat16 here? It looks contradictory to this PR.

Reply:
No special reason, changed to ```half```.